doc/source/tune/guides/plot_tune-distributed.rst

```diff
@@ -230,10 +242,12 @@ To summarize, here are the commands to run:
     # wait a while until after all nodes have started
     ray kill-random-node tune-default.yaml --hard
 
-You should see Tune eventually continue the trials on a different worker node. See the `Save and Restore <tune-usage.html#save-and-restore>`__ section for more details.
+You should see Tune eventually continue the trials on a different worker node. See the :ref:`Fault Tolerance <tune-fault-tol>` section for more details.
 
 You can also specify ``tune.run(upload_dir=...)`` to sync results with a cloud storage like S3, persisting results in case you want to start and stop your cluster automatically.
```

Comment:
Does this sentence make sense?
"sync results with a cloud storage like S3, persisting results in case you want"